apps/dsp-app/cypress/support/commands/ontology-command.ts

```diff
@@ -0,0 +1,27 @@
+import { faker } from '@faker-js/faker';
```

Comment:
can't we add it with api?

Reply:
same as with data-model-class-command. Should simulate user.